assembl/views/__init__.py

```diff
@@ -333,6 +333,13 @@ def process_export_list(ls):
     theme_name, theme_relative_path = get_theme_info(discussion)
     node_env = os.getenv('NODE_ENV', 'production')
     under_test = bool(config.get('under_test') or False)
+    # Define flag for testing environment
+    wsginame = config.get('wsginame') or None
+    if wsginame:
+        # wsginame values are defined in *.rc files
+        inject_bugherd = True if wsginame in ('preprod.wsgi')
```

Comment:
wsginame in ('preprod.wsgi')
2 cases:
1) substring in string ==> parentheses are not needed here
2) string in list/tuple of string ==> you need to add a comma here ('preprod.wsgi', ) or wsginame == 'preprod.wsgi'